src/arm/arm.c

```diff
@@ -209,58 +209,9 @@ static inline void ARMStep(struct ARMCore* cpu) {
 	LOAD_32(cpu->prefetch[1], cpu->gprs[ARM_PC] & cpu->memory.activeMask, cpu->memory.activeRegion);
 
 	unsigned condition = opcode >> 28;
-	if (condition != 0xE) {
-		bool conditionMet = false;
-		switch (condition) {
-		case 0x0:
-			conditionMet = ARM_COND_EQ;
-			break;
-		case 0x1:
-			conditionMet = ARM_COND_NE;
-			break;
-		case 0x2:
-			conditionMet = ARM_COND_CS;
-			break;
-		case 0x3:
-			conditionMet = ARM_COND_CC;
-			break;
-		case 0x4:
-			conditionMet = ARM_COND_MI;
-			break;
-		case 0x5:
-			conditionMet = ARM_COND_PL;
-			break;
-		case 0x6:
-			conditionMet = ARM_COND_VS;
-			break;
-		case 0x7:
-			conditionMet = ARM_COND_VC;
-			break;
-		case 0x8:
-			conditionMet = ARM_COND_HI;
-			break;
-		case 0x9:
-			conditionMet = ARM_COND_LS;
-			break;
-		case 0xA:
-			conditionMet = ARM_COND_GE;
-			break;
-		case 0xB:
-			conditionMet = ARM_COND_LT;
-			break;
-		case 0xC:
-			conditionMet = ARM_COND_GT;
-			break;
-		case 0xD:
-			conditionMet = ARM_COND_LE;
-			break;
-		default:
```

Comment:
The 0xF case that you're implicitly removing here is needed on the medusa branch.